diff --git a/node_modules/nativefier/lib/options/optionsMain.js b/node_modules/nativefier/lib/options/optionsMain.js
index 2d1d01a..480c395 100644
--- a/node_modules/nativefier/lib/options/optionsMain.js
+++ b/node_modules/nativefier/lib/options/optionsMain.js
@@ -59,6 +59,7 @@ async function getOptions(rawOptions) {
             overwrite: rawOptions.overwrite,
             platform: rawOptions.platform,
             portable: (_g = rawOptions.portable) !== null && _g !== void 0 ? _g : false,
+            quiet: true,
             targetUrl: rawOptions.targetUrl === undefined
                 ? '' // We'll plug this in later via upgrade
                 : normalizeUrl_1.normalizeUrl(rawOptions.targetUrl),
@@ -137,7 +138,7 @@ async function getOptions(rawOptions) {
         log.debug('Running in verbose mode! This will produce a mountain of logs and', 'is recommended only for troubleshooting or if you like Shakespeare.');
     }
     else {
-        log.setLevel('info');
+        log.setLevel('silent');
     }
     if (options.packager.electronVersion) {
         const requestedVersion = options.packager.electronVersion;
